### PR TITLE
fix(examples): updated examples to be compliant with latest API version

### PR DESCRIPTION
**Pull request by dmlittle**
What: Updates examples to be up to date with the latest API changes.

### CODE REVIEW

**Comment by dmlittle** [comment]
@mgartner @elnaz @brianseitel @neelk07 
Updated Java examples

File: lob-java-examples/src/main/java/com/lob/examples/CheckExample.java [hunk 1]
```diff
@@ -48,9 +49,19 @@ public static void main(final String[] args) throws Exception {
 
         printResponse("Bank Account Response", exampleBankAccountResponse);
 
+        // Verify a the previously created bank account
+        final BankAccountVerifyRequest exampleBankAccountVerifyRequest = BankAccountVerifyRequest.builder()
+                .id(exampleBankAccountResponse.getId())
+                .amounts(25, 63)
+                .build();
```
Comment (elnaz) [inline lob-java-examples/src/main/java/com/lob/examples/CheckExample.java]: Looks like this has one indent too many?

**Comment by elnaz** [comment]
LGTM. Thanks for saving the day @dmlittle !

**Comment by javadev** [comment]
Good, examples were improved. The version for examples' pom.xml may be changed with 2.0.2.

**Comment by dmlittle** [comment]
@elnaz fixed the indentation.
@javadev version has been changed to 2.0.2 already :+1:

**Comment by javadev** [comment]
LGTM

**Comment by mgartner** [comment]
LGTM

**Pull request merged**